ga4gh/frontend.py

```diff
@@ -23,8 +23,46 @@
 app = flask.Flask(__name__)
 
 
-class ServerStatus(object):
+class Version(object):
+    """
+    A major/minor/revision version tag
+    """
+    @classmethod
+    def parseString(cls, versionString):
+        versions = versionString.strip('vV').split('.')
+        return Version(*versions)
+
+    def __init__(self, major, minor, revision):
```

Comment:
Python uses a tuple for this of the form `(major, minor, revision)`. The advantage then is comparisons (<, ==, etc) are for free. Do we gain much from this class? Where do we use it?